src/libraries/Common/src/Interop/OSX/System.Native/Interop.Log.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
You put this under src/Interop/OSX.  Is that correct?  Or should there be an iOS-specific folder?

Reply:
Sadly there is no general mac folder where this would be better placed (https://developer.apple.com/documentation/foundation/1395275-nslog) or is the intention to copy the interop to System.Native implementation for all OSes?

Reply:
> Sadly there is no general mac folder where this would be better placed

Meaning this API is available on any mac, whether OSX or iOS or watch or whatever?  We can certainly create a folder for such APIs, assuming we're also making sure that the export it's targeting from System.Native is built on all such platforms.  Or if we're only building in the export for iOS, then an iOS-specific folder would make more sense.  Presumably in the fullness of time we'll find some APIs that are specific to iOS and so we'll want an iOS folder anyway?

Reply:
I'd move it to iOS honestly (just like in my previous commit) we are not going to use this API on OSX, only iOS/tvOS/watchOS which all are basically the same OS almost 🙂

Reply:
It's possible that we find some API which will be ios only but at the level, we operate it's most likely that PAL for ios, tvos, watchos, catalyst and some parts macOS will be identical

Reply:
I'd use something generic like `apple` for the pieces that work across mac/ios/tvos/watchos.

Doesn't need to be done here though since there are a lot of cases where we'll need to rename the Interop/OSX folder.

Reply:
so are we ok with OSX then?

Reply:
I think so. It does not make sense to try to redo the directory structure in this PR.